test/uk/gov/hmrc/hmrcemailrenderer/templates/TemplateLocatorSpec.scala

```diff
@@ -501,6 +501,8 @@ class TemplateLocatorSpec extends UnitSpec with OneAppPerSuite {
         "customs_financials_new_postponed_vat_notification",
         "cgtpd_email_verification",
         "tdq_compliance_all_required_headers_missing",
+        "tdq_compliance_partially_complaint_invalid_or_missing_connection_method",
```

Comment:
I think it should be `tdq_compliance_partially_compliant_invalid_or_missing_connection_method`

Reply:
good catch, thanks! Addresse